src/main.rs

```diff
@@ -110,13 +109,13 @@ async fn main() -> Result<()> {
             )?;
 
             actors.push(actor);
-            error_futures.push(Box::pin(async move { error_ch.recv().await }));
+            error_channels.push(error_ch);
```

Comment:
Good call switching to channels here; I think there's actually a spinner in the old code (left side line 154) that this resolves.

Reply:
:+1: